CHANGELOG.md

```diff
@@ -81,6 +81,10 @@ BUG FIXES:
 
 ## 2.5.0 (March 30, 2020)
 
+BREAKING CHANGES:
```

Comment:
:thinking: if we should use `REGRESSION` as they were not intentional breaking changes and if I understand root cause is that GHE server versions were quite a bit behind a github.com and once they are on new enough versions they should work.

Reply:
It was broken in a minor version instead of a major number.

Reply:
I hear you and I agree with the sentiment but it's not an intentional change here, which is what `BREAKING CHANGES` typically means. Even in the upstream API this happened because github was not consistent with its api release across products. It's absolutely a bug and I would :+1:  `REGRESSION` or `BUG` as it's more accurate and does not imply intent. I am a maintainer on many projects that use semver very strictly, I weigh such breaking decisions heavily. This implies that the author and reviewer knew about it and decided it was a good idea anyways.

Reply:
I like the option to use `REGRESSION`.  The distinction between the two makes sense, thanks for the guidance @majormoses.

```suggestion
REGRESSION:
```